docs/_docs/persistence/change-data-capture.adoc

```diff
@@ -171,6 +171,28 @@ For example, CDC was turned off several times: `000000000000002.wal`, `000000000
 Then, after the command is executed, the following segment links will be deleted: `000000000000002.wal`, `000000000000003.wal`, `000000000000008.wal`.
 The application will start from the `0000000000000010.wal` segment after being enabled.
 
+== Forcefully resend all cache data to CDC
+
+When the CDC has been forcefully disabled for a while, cache changes will be skipped.
+In this case it is necessary to resend data from existing caches. For example, this is important for the following cases:
+
+- pre-synchronizing replication;
+- handling missed segments.
+
+WARNING: There are no guarantees of handling concurrent updates.
+
+NOTE: The command will be canceled if cluster was not rebalanced or topology changed (node left/joined, baseline changed).
+
+To forcefully resend all cache data to CDC you can run the following link:tools/control-script[Control Script] command:
+
+[source,shell]
+----
+# Forcefully resend all cache data to CDC. Iterates over caches and writes data entries to the WAL to get captured by CDC:
+control.sh|bat --cdc resend --caches cache1,...,cacheN
+----
+
+The command will iterate over caches and writes data entries to the WAL to get captured by the CDC application.
```

Comment:
Do we still resend entries only for primary partitions?
It worth to be mentioned in doc.